ansible_collections/arista/avd/examples/l2ls-fabric/README.md

```diff
@@ -0,0 +1,413 @@
+# Example for L2LS Fabric
+
+## Introduction
+
+This example includes and describes all the AVD files used to build a Layer 2 Leaf Spine (L2LS) Fabric, with the following nodes:
+
+- 2 Spine nodes
+- 4 Leaf nodes
+
+The network fabric in this example is purely Layer 2. Routing is handled by an external Firewall/L3 Device. Later, in this example, we will discuss adding L3 routing to the Spines. For now, we will focus on defining the fabric variables to build out this L2LS Topology. Before we get started, we need to ensure that we have installed AVD, with necessary requirements, which are covered in the Installation & Requirements section.
+
+The example is meant as a starting foundation. You may build more advanced fabrics based on this design. To keep things simple, the Arista eAPI will be used to communicate with the switches.
+
+???+ info
+
+    The configurations may also be applied with CloudVision with a few updates to your playbook and Ansible variables.
+
+## Installation & Requirements
+
+1. Install AVD - Installation guide found [here](../../docs/installation/collection-installation.md).
+2. Install Ansible module requirements - Instructions found [here](../../docs/installation/requirements.md).
+3. Run the following playbook which will copy Getting Started examples to your current working directory.
+
+```bash
+# current working directory: ~/ansible-avd-examples
+ansible-playbook arista.avd.install_examples
+```
+
+Output will show something similar to the following. If not, please make sure that AVD and all requirements are installed properly. Also verify that the collections_paths in the ansible.cfg file is correct.
```

Comment:
```suggestion
The output will show something similar to the following. If not, please ensure that AVD and all requirements are correctly installed. Also, verify the collections_paths in the ansible.cfg are correct.
```

Reply:
Why are we talking about `collections_paths` when our advised installation method is using ansible-galaxy - hence using default directories.

Reply:
@ClausHolbechArista Agreed that default paths should be used, but someone may modify the ansible.cfg and change collection_paths to something else.  So I thought it was prudent to have them check it out.  I am fine removing it...

Reply:
@ClausHolbechArista I removed the reference to collection_paths.

Reply:
@JulioPDX updated with your suggestions.